omod/src/main/java/org/openmrs/module/spa/servlet/SpaServlet.java

```diff
@@ -55,10 +82,29 @@ protected long getLastModified(HttpServletRequest servletRequest) {
 
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
-        if (SpaModuleUtils.isRemoteAssetsEnabled()) {
-            handleRemoteAssets(request, response);
+        String requestURI = request.getRequestURI();
+        if (requestURI.contains("/config")) {
```

Comment:
by default, all `GET` requests return an `index.htm` file by default.  This is why i think i manually need to call the corresponding controllers